documentation/modules/upgrading/con-upgrade-older-clients.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// assembly-upgrade-zookeeper.adoc
+
+[id='con-upgrade-older-clients-{context}']
+= Upgrading clients with older message formats
+
+[role="_abstract"]
+Before Kafka 3.0, you would configure a specific message format for brokers using the `log.message.format.version` property (or the `message.format.version` property at the topic level). 
+This allowed brokers to support older Kafka clients that were using an outdated message format. 
```

Comment:
I think this is misleading as even without this field the older client are still supported.

Reply:
I've rewritten as follows:

> Before Kafka 3.0, you could configure a specific message format for brokers using the `log.message.format.version` property (or the `message.format.version` property at the topic level). 
> This allowed brokers to accommodate older Kafka clients that were using an outdated message format. 
> Though Kafka inherently supports older clients without explicitly setting this property, brokers would then need to convert the messages from the older clients, which came with a significant performance cost.